cnv/cnv_virtual_machines/cnv_vm_networking/virt-using-mac-address-pool-for-vms.adoc

```diff
@@ -7,7 +7,7 @@ toc::[]
 :FeatureName: KubeMacPool
 include::modules/technology-preview.adoc[leveloffset=+1]
 
-The _KubeMacPool_ component provides a MAC address pool service for virtual machines in designated namepsaces. Enable a MAC address pool in a namespace by applying the KubeMacPool label to that namespace.
+The _KubeMacPool_ component provides a MAC address pool service for virtual machines in designated namespaces. Enable a MAC address pool in a namespace by applying the KubeMacPool label to that namespace.
```

Comment:
I think we need to emphasize that kmp only serves secondary nics. 
I saw you added it later - but it needs to be clear in the first introduction line (similar to how it is in [u/s documentation](The _KubeMacPool_ component provides a MAC address pool service for virtual machines in designated namespaces. Enable a MAC address pool in a namespace by applying the KubeMacPool label to that namespace.). something like:

```
The _KubeMacPool_ component provides a MAC address pool service for additional NICs in virtual machines in designated namespaces. Enable a MAC address pool in a namespace by applying the KubeMacPool label to that namespace.
```